kustomize/base/7.1.0/ingress/ingress.yaml

```diff
@@ -16,60 +15,99 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-buffers: 4 256k
     nginx.ingress.kubernetes.io/proxy-busy-buffers_size: 256k
     nginx.ingress.kubernetes.io/error-log-level: "debug"
+  name: mtls
+spec:
+  rules:
+    - host: $(MTLS_FQDN)
+      http:
+        paths:
+          - backend:
+              service:
+                name: ig
+                port:
+                  number: 80
+            path: /am/oauth2/realms/root/realms/alpha/access_token
+            pathType: Exact
+          - backend:
+              service:
+                name: ig
+                port:
+                  number: 80
+            path: /am/oauth2/realms/root/realms/alpha/register
+            pathType: Exact
+          - backend:
+              service:
+                name: ig
+                port:
+                  number: 80
+            path: /rs/
+            pathType: Prefix
+  tls:
+    - hosts:
+        - $(MTLS_FQDN)
+      secretName: obwac
+---
+# Ingress for access to routes protected with TLS by a ForgeRock cert
```

Comment:
Probably a LetsEncrypt cert. Just a TLS cert rather than MTLS? Cert backed by browser trusted cert issuers?

Reply:
Yes, that's what I was referring to. Didn't want to put let's encrypt incase we changed issuer.